lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -541,7 +557,9 @@ def main():
             port=dict(type='int', default=5432),
             unix_socket=dict(default='', aliases=['login_unix_socket']),
             login=dict(default='postgres', aliases=['login_user']),
-            password=dict(default='', aliases=['login_password'], no_log=True)
+            password=dict(default='', aliases=['login_password'], no_log=True),
+            ssl_mode=dict(default='disable'),
+            ssl_rootcert=dict(default='')
```

Comment:
postgres_db.py uses `""` for its defaults, postgres_privs.py uses `''` and postgres_user.py uses `None`. This PR tries to adhere to formatting.styles of each file (which differs between each other...).

Postgres itself does not mind either way.